source/develop/developer-guide/vdsm/developers.html.md

```diff
@@ -18,12 +18,19 @@ wiki_last_updated: 2015-08-26
 To build VDSM, enable the oVirt repositories by installing the ovirt-release rpm:
 
       yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
+  or if you are using fedora 
+      /usr/bin/dnf install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
 
 If you need a previous installation, use the corresponding repository instead:
 
       yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release40.rpm 
       yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release36.rpm 
       yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release35.rpm
+  for fedora 
```

Comment:
Thanks for the patch!

Please add new lines before and after "for fedora" so the markdown will be rendered correctly.
Also please use Fedora instead of fedora